tests/document_ai/test_classify.py

```diff
@@ -0,0 +1,71 @@
+import os
+import unittest
+
+from tensorlake.documentai import DocumentAI
+from tensorlake.documentai.models import PageClassConfig, ParseStatus
```

Comment:
Can you make sure PageClassConfig, ParseStatus is in __init__.py of documentai ?